src/main/resources/jenkins/plugins/logstash/configuration/Logzio/config.jelly

```diff
@@ -0,0 +1,9 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+      <f:entry title="${%Logz.io Host}" field="host">
+        <f:textbox default="https://listener.logz.io:8071"/>
+      </f:entry>
+      <f:entry title="${%Logz.io key}" field="key">
```

Comment:
In the help you explain it to be the account token so why not directly use this in the field label